+++
abstract = "Genome-scale models of metabolism can illuminate the molecular basis of cell phenotypes. Since some enzymes are only active in specific cell types, several algorithms use omics data to construct cell-line- and tissue-specific metabolic models from genome-scale models. However, these methods are often not rigorously benchmarked, and it is unclear how algorithm and parameter selection (e.g., gene expression thresholds, metabolic constraints) affects model content and predictive accuracy. To investigate this, we built hundreds of models of four different cancer cell lines using six algorithms, four gene expression thresholds, and three sets of metabolic constraints. Model content varied substantially across different parameter sets, but the algorithms generally increased accuracy in gene essentiality predictions. However, model extraction method choice had the largest impact on model accuracy. We further highlight how assumptions during model development influence model prediction accuracy. These insights will guide further development of context-specific models, thus more accurately resolving genotype-phenotype relationships."
abstract_short = "This study presents a comparative analysis of hundreds of models built for four different cancer cell lines using several prominent model extraction algorithms. The evaluation provides insights into how their assumptions affect the prediction capabilities of contextspecific models."
authors = ["Sjoerd Opdam","Anne Richelle","Benjamin Kellman","Shanzhong Li","Daniel C. Zielinski","Nathan E. Lewis"]
date = "2017-04-07"
image_preview = ""
math = true
publication_types = ["1"]
publication = "In *Cell Systems*, Elsevier."
publication_short = "In *Cell Systems*"
selected = true
title = "A Systematic Evaluation of Methods for Tailoring Genome-Scale Metabolic Models"
#url_code = "https://github.com/bkellman/NEC_HMO"
#url_dataset = "#"
url_pdf = "https://bkellman.github.io/img/Opdam_Cell_Systems_2017.pdf"
#url_project = "https://bkellman.github.io/project/nec-hmo/"
#url_slides = "#"
#url_video = "#"

#[[url_custom]]
#name = "Custom Link"
#url = "/img/gutjnl-2016-312819.full"

# Optional featured image (relative to `static/img/` folder).
[header]
image = "headers/bubbles-wide.jpg"
#caption = "My caption :smile:"

#More detail can easily be written here using *Markdown* and $\rm \LaTeX$ math code.

+++

